hack/make/images.mk

```diff
@@ -0,0 +1,28 @@
+MASTER_IMAGE ?= quay.io/dynatrace/dynatrace-operator:snapshot
+
+# Default bundle image tag
+BUNDLE_IMG ?= controller-bundle:$(VERSION)
+BRANCH_IMAGE ?= quay.io/dynatrace/dynatrace-operator:snapshot-$(shell git branch --show-current | sed "s/[^a-zA-Z0-9_-]/-/g")
```

Comment:
could `$(shell git branch --show-current | sed "s/[^a-zA-Z0-9_-]/-/g")` be moved to a variable (not named `TAG`) and reused?